client/src/main/kotlin/no/nav/su/se/bakover/client/oppdrag/utbetaling/UtbetalingRequest.kt

```diff
@@ -103,16 +103,18 @@ data class UtbetalingRequest(
         val saksbehId: String,
         /** Fødselsnummer eller Organisasjonsnummer [9,11] tegn */
         val utbetalesTilId: String,
+        /** [0,30] tegn - en referanse til hvilken utbetaling-id (vår) utbetalingslinjen er koblet til */
+        @field:JacksonXmlProperty(localName = "henvisning")
+        val utbetalingId: String,
         /** Makslengde 30 tegn */
+        @field:JacksonXmlProperty(localName = "refDelytelseId") // TODO vurder prop order, virker som rekkefølgen er avhengig av om feltet er annotert eller ei.
```

Comment:
Dette bør kunne verifiseres i en test ganske enkelt?

Reply:
Ja, testen fanger det opp.. men orket ikke fikse order for alle props før jeg vet om det er problemet.